presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -212,6 +213,11 @@ public SystemSessionProperties(
                         "Use grouped execution for aggregation when possible",
                         featuresConfig.isGroupedExecutionForAggregationEnabled(),
                         false),
+                booleanProperty(
+                        GROUPED_EXECUTION_FOR_ALL_ELIGIBLE_TABLE_SCANS,
+                        "Use grouped execution for all eligible table scans",
```

Comment:
nit: `Experimental: use grouped execution for all eligible table scans` :)

Reply:
if it's experimental, should the config property be prefixed with `experimental.`?